src/js/modules/StaticMap.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react';
+
+class StaticMap extends Component {
+
+  getLocation() {
+
+    const { street, city, state, zip, latitude, longitude } = this.props.location;
+
+    if (street) {
+      return encodeURIComponent(`${street} ${city || "Austin"}, ${ state || "Texas"} ${zip}`);
+    }
+
+    if (latitude && longitude) {
+      return encodeURIComponent(`${latitude}, ${longitude}`);
+    }
+
+    return encodeURIComponent("City Hall, Austin, Texas 78701");
```

Comment:
Could we maybe save this string as a const called something like `fallbackAddress`? Or just something to make it clear in for others?

Reply:
yep